neural_compressor/adaptor/torch_utils/smooth_quant.py

```diff
@@ -1205,6 +1208,8 @@ def trace(self, model, dummy_input):
         optimize_numerics = False
         if hasattr(model, "device"):
             orig_device = model.device
+            if hasattr(model, "type"):
```

Comment:
model.device, 'type'? BTW, how to handle the index if cuda:0/1

Reply:
orig_device will be "cuda:0/1", if the origin_device != cpu, the tensor will convert to cpu tensor.